docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+
+services:
+
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
Remember this starts up only a development server. 

Try using gunicorn instead: https://docs.djangoproject.com/en/2.2/howto/deployment/wsgi/gunicorn/

Reply:
I'm add `prod` configuration later on